.bazelignore

```diff
@@ -1,9 +1,9 @@
-# Some npm modules include their own WORKSPACE, so we need to ignore them from
-# our build.
-node_modules
+# Some npm modules include their own WORKSPACE/BUILD files, so we need to ignore
+# them from our build. Manually listing them is not ideal, but we can't ignore
+# the entire node_modules folder, because we need it for our bazel builds.
+node_modules/@tensorflow/tfjs-core/src
```

Comment:
Would ignoring `**/BUILD` not be preferable?

Reply:
It didn't seem to work...